Files/Views/Pages/PropertiesDetails.xaml.cs

```diff
@@ -1,41 +1,31 @@
 ﻿using Files.Filesystem;
-using Files.View_Models;
 using Files.View_Models.Properties;
 using Microsoft.Toolkit.Uwp.Helpers;
 using System;
 using System.Diagnostics;
 using System.Threading.Tasks;
 using Windows.ApplicationModel.Core;
-using Windows.Storage;
 using Windows.UI.Xaml;
 using Windows.UI.Xaml.Controls;
 using Windows.UI.Xaml.Navigation;
 
-// Il modello di elemento Pagina vuota è documentato all'indirizzo https://go.microsoft.com/fwlink/?LinkId=234238
-
 namespace Files
 {
-    /// <summary>
-    /// Pagina vuota che può essere usata autonomamente oppure per l'esplorazione all'interno di un frame.
-    /// </summary>
-    public sealed partial class PropertiesDetails : Page
+    public sealed partial class PropertiesDetails : PropertiesTab
     {
-        public FileProperties BaseProperties { get; set; }
-
-        public SelectedItemsPropertiesViewModel ViewModel { get; set; }
-
         public PropertiesDetails()
         {
             this.InitializeComponent();
         }
 
         private void Properties_Loaded(object sender, RoutedEventArgs e)
```

Comment:
While we are at it I believe this should be defined as `protected override` and the base method in `PropertiesTab.cs` should be marked as `protected virtual`.